README.md

```diff
@@ -51,6 +51,13 @@ This repo contains everything needed to run a musicbrainz slave server with sear
 * Linux or macOS
   (Windows is not documented yet, it is recommended to use Ubuntu via VirtualBox instead.)
 
+If you use Ubuntu 19.10 or later, the above requirements can be set up by running:
+```bash
+sudo apt-get update && \
+sudo apt-get install docker.io docker-compose git && \
+sudo systemctl enable --now docker.service
```

Comment:
You are sure this is not activated automatically? Normally it should be.

Reply:
I’m not sure but it seems safer to enable it in either case and it tells newcomers there is a service running at system level behind the scenes.

Reply:
If it is automatically activated and started we don't need to do it again. It is just redundant.
And I don't think that newcomers should use this project anyway or they don't care if a daemon is running or not.